src/state/stores/photo.ts

```diff
@@ -28,17 +51,3 @@ async function setPhoto(searchTerms) {
     };
   });
 }
-
-async function load() {
-  if (!isBrowser) {
-    return;
-  }
-
-  const settingsStore = await getSettingsStore();
-  settingsStore.subscribe((settings) => {
-    const { searchTerms } = settings;
-    setPhoto(searchTerms);
-  });
-}
-
-load();
```

Comment:
Calling load here and not exporting this was not allowing for waiting explicit setup, as well as `manager` being initialized at the top. Refactoring `load` into `setup` and including the setup of the manager instead of letting it float in module scope seemed like the best way to go.